api/templates/server/operation.gotmpl

```diff
@@ -1,8 +1,7 @@
 package {{ .Package }} // import "github.com/docker/docker/api/types/{{ .Package }}"
 
 // ----------------------------------------------------------------------------
-// DO NOT EDIT THIS FILE
-// This file was generated by `swagger generate operation`
+// Code generated by `swagger generate operation` DO NOT EDIT.
```

Comment:
This is purely aesthetical, but can we please add a point or a comma before DO NOT EDIT?

As in
```go
// Code generated by `swagger generate operation`. DO NOT EDIT.
```